### PR TITLE
Use bootstrap dialog in place of standard confirm

### PR DESCRIPTION
**Pull request by ravicodder**
### What is this PR for?
Use bootstrap dialog in place of standard confirm

### What type of PR is it?
 Improvement 

### What is the Jira issue?
NA
### Screenshots (if appropriate)
Before:
 
screen shot 2016-04-14 at 4 16 06 pm

After:
screen shot 2016-04-14 at 4 15 24 pm

### Questions:
* Does the licenses files need update?no
* Is there breaking changes for older versions?no
* Does this needs documentation?no

### CODE REVIEW

**Comment by Leemoonsoo** [comment]
Thanks for the improvement.

There're a [test case](https://github.com/apache/incubator-zeppelin/blob/master/zeppelin-server/src/test/java/org/apache/zeppelin/integration/ZeppelinIT.java#L208) that depends standard confirm.

CI fails with 

```
Results :

Tests in error: 
  ZeppelinIT.testSparkInterpreterDependencyLoading:208 Â» NoAlertPresent No alert...

Tests run: 14, Failures: 0, Errors: 1, Skipped: 0
```

Could you update this test, too?

**Comment by prabhjyotsingh** [comment]
you need to fix it https://github.com/apache/incubator-zeppelin/pull/837/files#diff-a7fb72a3220c2a1f756a04eaa5cf1dc6R245 as well

**Comment by ravicodder** [comment]
@Leemoonsoo Thanks for review, Test updated .
@prabhjyotsingh Thanks for help.

File: zeppelin-web/src/app/interpreter/interpreter.controller.js [hunk 1]
```diff
@@ -60,44 +60,48 @@ angular.module('zeppelinWebApp').controller('InterpreterCtrl', function($scope,
...
+          }
+          if (setting.depArtifact !== '' || setting.depArtifact) {
+             $scope.addNewInterpreterDependency(settingId);
+          }
+          // add missing field of option
+          if (!setting.option) {
+             setting.option = {};
+          }
+          if (setting.option.remote === undefined) {
+                      // remote always true for now
```
Comment (felixcheung) [inline zeppelin-web/src/app/interpreter/interpreter.controller.js]: could you pls fix the indentation?

**Comment by felixcheung** [comment]
thx. any more comment?

**Comment by Leemoonsoo** [comment]
@ravicodder Do you have JIRA id?

**Comment by ravicodder** [comment]
@Leemoonsoo  No JIRA id for this.

**Comment by felixcheung** [comment]
My bad. Are you asking if this issue should have a JIRA to track this change?

**Comment by Leemoonsoo** [comment]
@ravicodder Also can i ask your user ID on [JIRA](https://issues.apache.org/jira/browse/ZEPPELIN)?

**Comment by ravicodder** [comment]
@Leemoonsoo  My JIRA user ID is ravicodder . Should I create a JIRA issue for this change?

**Comment by Leemoonsoo** [comment]
@ravicodder i was asking JIRA user ID for ZEPPELIN-818, somehow i commented here by mistake. Having JIRA issue is not mandatory but i think it's always good to have one. For example JIRA issue can provide additional information about the change, like 'fix version'. Also release note used to include list of JIRA issues, not Pullrequests.

**Pull request closed without merging**